Init x-beta documentation (#148)
bump-sh/docs: src/_help/doc-beta.md

The `x-beta` property is a boolean.

Adding or removing this property is a strucutal change.
A change on a beta component, can't be breaking.

## Example usage

Here under is an example of a beta Operation, a beta request body and a beta Schema attribute.

```yaml
  "paths": {
    "/diffs": {
      "post": {
        "x-beta": true,
        "description": "Create a diff between any two given API definitions",
        "requestBody": {
          "description": "The diff creation request object",
          "content": {
            "application/json": {
              "schema": {
                "x-beta": true,
                "type": "object",
                "properties": {
                  "url": {
                    "x-beta": true,
                    "type": "string",
                    "format": "uri",
                    "description": "**Required** if `definition` is not present.\nCurrent definition URL. It should be accessible through HTTP by Bump.sh servers.\n"
                  },
                  ...
                }
              }
            }
          }
        }
      }
    }
  }
```

> The documentation displays a "Beta" flag on the components:

Reviewer: You don't like the british curly quotes? They are the common topographic characters for English quotes :)
```suggestion
The documentation displays a “Beta” flag on the components:
```

> The documentation displays a "Beta" flag on the components:

Reviewer: Conversation is resolved but I still see the straight double quotes :sob: 

```suggestion
The documentation displays a “Beta” flag on the components:
```

Author: I have not pushed yet

Author: Ok, I've pushed.

---
title: Beta
---

- TOC
{:toc}

Use the `x-beta` property inside an Operation, a schema or a parameter object to identify it as beta.
The `x-beta` property is a boolean.

> Add or remove this property is a strucutal change.

Author: Oh ! It's Canadian english...

A change on a beta component, can't be breaking.

## Example usage

> The following screen capture shows the rendering of adding two code samples `cURL` and `Ruby` to your Operation.
> 

Reviewer: This is a bad copy-pasta. Can you remove the line?

```suggestion
```

Author: Indeed. Sorry

> The following screen capture shows the rendering of adding two code samples `cURL` and `Ruby` to your Operation.
> 
> Hereunder is an example of a beta Operation, Schema and Parameter.

Reviewer: ```suggestion
Here under is an example of a beta Operation, a beta request body and a beta Schema attribute.
```

```yaml
  "paths": {
    "/diffs": {
      "post": {
        "x-beta": true,
        "description": "Create a diff between any two given API definitions",
        "requestBody": {

>           "x-beta": true,

Reviewer: Did we add the parsing of `x-beta` on Request body objects? I thought we only added it on `Operations` and `Schema`?

Author: Yes, my bad, you're right. I forgot to correct this.

          "content": {
            "application/json": {
              "schema": {
                "x-beta": true,
                "type": "object",
                "properties": {
                  "url": {
                    "x-beta": true,
                    "type": "string",
                    "format": "uri",
                    "description": "**Required** if `definition` is not present.\nCurrent definition URL. It should be accessible through HTTP by Bump.sh servers.\n"
                  },
                  ...
                }
              }
            }
          }
        }
      }
    }
  }
```

> The documentation display a "Beta" flag on the components:

Reviewer: ```suggestion
The documentation displays a “Beta” flag on the components:
```
